packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -312,8 +312,9 @@ abstract contract StandardBridge is Initializable {
         } catch {
             // Something went wrong during the bridging process, return to sender.
             // Can happen if a bridge UI specifies the wrong L2 token.
-            // We reverse both the local and remote token addresses, as well as the to and from
-            // addresses. This will preserve the accuracy of accounting based on emitted events.
+            // We reverse the to and from addresses to make sure the tokens are returned to the
+            // sender on the other chain and preserve the accuracy of accounting based on emitted
+            // events.
```

Comment:
This was wrong (my mistake). 
The ordering of the token addresses is unchanged.

https://github.com/ethereum-optimism/optimism/blob/772cea0fb34017a9a2dd3edacfa8077414c093a5/packages/contracts-bedrock/contracts/universal/StandardBridge.sol#L475-L477